### PR TITLE
Get go from buster-backports instead of website

### PR DESCRIPTION
**Pull request by rasmus-hagberg-debricked**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google-chrome.list \
 # Need backports for openjdk 11 package
     && echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/backports.list \
```
Comment (rasmus-hagberg-debricked) [inline Dockerfile:20]: openjdk-11-jdk is available in stable. I don't think we use stretch-backports for anything, and we should be able to remove it

Comment (sweoggy) [reply Dockerfile:20]: Remove it and see if the build still passes :)

File: Dockerfile [hunk 2]
```diff
@@ -141,18 +143,12 @@ RUN cd / \
...
-    && curl -L -O https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz \
-    && tar -xvf go${GO_VERSION}.linux-amd64.tar.gz \
-    && mv go ${BIN_DIRECTORY}/golang \
-    && rm go${GO_VERSION}.linux-amd64.tar.gz \
-    && ln -s ${BIN_DIRECTORY}/golang/bin/go ${BIN_DIRECTORY}/go
-ENV GOROOT ${BIN_DIRECTORY}/golang
-ENV PATH ${GOROOT}/bin:$PATH
+
+# Install python and pip and related dev packages.
+RUN apt update && apt install python3 python3-dev python3-pip python3-venv libffi-dev libssl-dev -y && pip3 install pipenv
```
Comment (sweoggy) [inline Dockerfile:148]:
```suggestion
RUN apt update && apt install python3 python3-dev python3-pip python3-venv libffi-dev libssl-dev -y \
    && pip3 install pipenv
```

Comment (sweoggy) [reply Dockerfile:148]: The line is quite long otherwise

**Pull request merged**